dotCMS/src/main/java/com/dotmarketing/cms/content/submit/action/SubmitContentAction.java

```diff
@@ -107,11 +107,6 @@ public ActionForward submitContent(ActionMapping rMapping, ActionForm form,
 		else
 			htmlServlet = referrer.substring(index + 1);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "htmlServlet". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)